util/topgen.py

```diff
@@ -1260,6 +1260,8 @@ def render_template(template_path: str, rendered_path: Path,
         ]
         for idx, path in enumerate(out_paths):
             # C Header + C File + Clang-format file
+            gencmd_c = warnhdr + GENCMD.format(top_name=top_name)
+            gencmd_bzl = gencmd_c.replace("//", "#")
```

Comment:
Is hat being used somewhere, or is this in preparation?

Reply:
It's used by the other PR but you are right that it should not be in that commit

Reply:
Actually not it's used further down in the file, I was confused

Reply:
Thanks!